pkg/parser/instance/container.go

```diff
@@ -34,16 +35,6 @@ func NewCommunityContainer(
 		},
 	}
 
-	imageSchema := schema.String("Docker Image to use.")
```

Comment:
Is there a reason to move this and other `imageSchema`'s below in their corresponding files?

Reply:
I've reverted the order in https://github.com/cirruslabs/cirrus-cli/pull/908 and the tests still pass.

The PR above explains why this happens.